### PR TITLE
Add extra labels to kubernetes-sigs/cluster-api

### PR DESCRIPTION
**Pull request by vincepri**
Signed-off-by: Vince Prignano <vincepri@vmware.com>

Fixes https://github.com/kubernetes-sigs/cluster-api/issues/991

### CODE REVIEW

File: label_sync/labels.yaml [hunk 1]
```diff
@@ -1175,3 +1175,35 @@ repos:
...
+        addedBy: anyone
+      - color: 0ffa16
+        description: Issues or PRs related to the data model
+        name: area/data-model
+        target: both
+        prowPlugin: label
+        addedBy: anyone
+      - color: 0ffa16
+        description: Issues or PRs related to the APIs
+        name: area/api
```
Comment (detiber) [inline label_sync/labels.yaml]: what is the difference between api and data-model here?

**Comment by ncdc** [comment]
Something for health/status?
Stability?

File: label_sync/labels.yaml (hunk 1 above)
Comment (vincepri) [reply label_sync/labels.yaml]: Maybe none, I heard someone during the call saying "api" and "data-model" labels, should I dedupe?

Comment (ncdc) [reply label_sync/labels.yaml]: +1, I'd say they're the same. I'd prefer keeping `area/api`

**Comment by ncdc** [comment]
@vincepri the CAPI issue this is for is https://github.com/kubernetes-sigs/cluster-api/issues/991

**Comment by ncdc** [comment]
Should we also have `area/testing`? The current test-related labels are:

- kind/failing-test
- needs-ok-to-test
- ok-to-test
- sig/testing
- kind/flake

**Comment by vincepri** [comment]
+1 to testing

File: label_sync/labels.yaml [hunk 2]
```diff
@@ -1175,3 +1175,62 @@ repos:
...
+        target: both
+        prowPlugin: label
+        addedBy: anyone
+      - color: 0ffa16
+        description: Issues or PRs related to health
+        name: area/health
+        target: both
+        prowPlugin: label
+        addedBy: anyone
+
```
Comment (detiber) [inline label_sync/labels.yaml]: whole lotta whitespace going on here, otherwise lgtm

Comment (vincepri) [reply label_sync/labels.yaml]: Fixed 😂

**Pull request merged**